Readme.md

```diff
@@ -1,27 +1,61 @@
+
+```js
+function OnValueChanged(s, e) {
+    $.ajax({
+        url: '@Url.Action("GridViewPartial", "Home")',
+        type: "GET",
+        data: { gridType: RadioButtonList.GetSelectedItem().text },
+        success: function (data) {
+            $('#dvContainer').html(data);
+        },
+        error: function (xhr, textStatus, errorThrown) {
+            alert('Request Status: ' + xhr.status + '; Status Text: ' + textStatus + '; Error: ' + errorThrown);
+        }
+    });
+}
+```
+
+```cs
+public ActionResult GridViewPartial(GridType gridType = GridType.Categories) {
+    IEnumerable model = null;
+
+    switch (gridType) {
+        case GridType.Categories:
+            model = NorthwindDataProvider.GetCategories();
+            ViewData["KeyFieldName"] = "CategoryID";
+            break;
+        case GridType.Products:
+            model = NorthwindDataProvider.GetProducts();
+            ViewData["KeyFieldName"] = "ProductID";
+            break;
+    }
+    // ...
+}
+```
+
+## Files to Review
+
+* [HomeController.cs](./CS/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/Controllers/HomeController.vb))
+* [GridViewPartial.cshtml](./CS/Views/Home/GridViewPartial.cshtml)
+* [Index.cshtml](./CS/Views/Home/Index.cshtml)
 
-This example is an MVC-based analog of the <a href="https://www.devexpress.com/Support/Center/p/E2968">E2968 - How to bind ASPxGridView with autogenerated columns to different data sources at runtime</a> code example. Note that in the case of MVC, implementation is much simpler. We use an AJAX callback to reload the GridView extension with the selected datasource. In addition, it is necessary to pass the datasource Id to GridView-specific callbacks. We use the approach described in the <a href="https://documentation.devexpress.com/#AspNet/CustomDocument9941">Passing Values to a Controller Action through Callbacks</a> help section for this purpose.<br><br><strong>See Also:</strong><br><a href="https://www.devexpress.com/Support/Center/p/E4063">E4063 - How to use the jQuery.ajax function with DevExpress MVC Extensions</a> <br><a href="https://www.devexpress.com/Support/Center/p/T191698">T191698 - How to isolate extension settings into separate helper class and share a single partial view to display multiple extensions</a>
+## Documentation
 
-<br/>
+* [Passing Values to a Controller Action through Callbacks](https://docs.devexpress.com/AspNetMvc/9941/common-features/callback-based-functionality/passing-values-to-a-controller-action-through-callbacks)
 
+## More Examples
 
+* [Grid View for ASP.NET Web Forms - How to bind a grid to different data sources at runtime](https://github.com/DevExpress-Examples/aspxgridview-created-at-runtime-switch-data-sources)
+* [Grid View for APS.NET MVC - How to use the jQuery.ajax function with DevExpress MVC extensions](https://github.com/DevExpress-Examples/mvc-gridview-jqueryajax-function)
```

Comment:
Note that "APS.NET" should be "ASP.NET". Can you update the actual title on the GitHub Readme page as well?

```suggestion
* [Grid View for ASP.NET MVC - How to use the jQuery.ajax function with DevExpress MVC extensions](https://github.com/DevExpress-Examples/mvc-gridview-jqueryajax-function)
```